index.html

```diff
@@ -554,6 +554,775 @@ <h2>OpenID Connect</h2>
+<a>verification method</a> `type` and its associated verification material
+format. For examples, see
+<a href="https://www.w3.org/TR/vc-di-ecdsa/">the Data Integrity ECDSA
+Cryptosuites</a> and <a href="https://w3c-ccg.github.io/lds-ed25519-2020/">
+the Data Integrity EdDSA Cryptosuites</a>. For a list of <a>verification
+method</a> types, please see the [[?SECURITY-VOCABULARY]].
+            </p>
+
+            <p>
+To increase the likelihood of interoperable implementations, this specification
+limits the number of formats for expressing verification material in a
+<a>controller document</a>. The fewer formats that implementers have to
+implement, the more likely it will be that they will support all of them. This
+approach attempts to strike a delicate balance between easing implementation
+and providing support for formats that have historically had broad deployment.
+            </p>
+
+            <p>
+A <a>verification method</a> MUST NOT contain multiple verification material
+properties for the same material. For example, expressing key material in a
+<a>verification method</a> using both `publicKeyJwk` and
+`publicKeyMultibase` at the same time is prohibited.
+            </p>
+
+            <p>
+An example of a <a>controller document</a> containing <a>verification
+methods</a> using both properties above is shown below.
+            </p>
+
+            <pre id="example-various-verification-method-types"
+              class="example nohighlight"
+              title="Verification methods using publicKeyJwk and publicKeyMultibase">
+    {
+      "@context": [
+        "https://www.w3.org/ns/did/v1",
+        "https://w3id.org/security/suites/jws-2020/v1",
+        "https://w3id.org/security/multikey/v1"
+      ]
+      "id": "did:example:123456789abcdefghi",
+      <span class="comment">...</span>
+      "verificationMethod": [{
+        "id": "did:example:123#_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A",
+        "type": "JsonWebKey", <span class="comment">// external (property value)</span>
+        "controller": "did:example:123",
+        "publicKeyJwk": {
+          "crv": "Ed25519", <span class="comment">// external (property name)</span>
+          "x": "VCpo2LMLhn6iWku8MKvSLg2ZAoC-nlOyPVQaO3FxVeQ", <span class="comment">// external (property name)</span>
+          "kty": "OKP", <span class="comment">// external (property name)</span>
+          "kid": "_Qq0UL2Fq651Q0Fjd6TvnYE-faHiOpRlPVQcY_-tA4A" <span class="comment">// external (property name)</span>
+        }
+      }, {
+        "id": "did:example:123456789abcdefghi#keys-1",
+        "type": "Multikey", <span class="comment">// external (property value)</span>
+        "controller": "did:example:pqrstuvwxyz0987654321",
+        "publicKeyMultibase": "z6MkmM42vxfqZQsv4ehtTjFFxQ4sQKS2w6WR7emozFAn5cxu"
+      }],
+      <span class="comment">...</span>
+    }
+            </pre>
+          </section>
+
+          <section>
+            <h3>Multikey</h3>
```

Comment:
Lets not refer to non standard key formats in TR track documents, we can refer to the work at IETF informatively though.